src/main/java/utils/StaticFileReader.java

```diff
@@ -0,0 +1,47 @@
+package utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+public class StaticFileReader {
+    private File file;
+
+    public StaticFileReader(File file) {
+        this.file = file;
+    }
+
+    public byte[] readAllBytes() {
+        try (FileInputStream inputStream = new FileInputStream(file)) {
+            byte[] buffer = new byte[(int) file.length()]; // 파일 길이만큼 바이트 배열 생성
+            int bytesRead = inputStream.read(buffer); // 파일 내용 읽기
+
+            if (bytesRead != file.length()) { // 잘못 변환된 경우 예외 처리
+                throw new IOException();
+            }
+            return buffer;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public String readAsStr() {
```

Comment:
```suggestion
    public String readAll() {
```
리턴 타입이 스트링이니 굳이 표시 안해 줘도 될 거 같아요.